blocks/product-comparison-table/product-comparison-table.css

```diff
@@ -188,7 +186,12 @@
     border-right: 1px solid #e4f2ff;
 }
 
-@media (min-width: 480px) {
+@media (min-width: 580px) {
+    .product-comparison-table-wrapper {
```

Comment:
Any reason for not dropping the table from the name - to be simply just `product-comparison` block ?

Reply:
I thought to put the table in the name, because this style was attached to a table like element, even though I didn't see any other place where the product comparison is done